main.js

```diff
@@ -160,4 +160,216 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+const availableStocks = inventory.map((availableStock) => {
+  return availableStock.originalStock - availableStock.sold;
+});
+
+function availableForSale(arr) {
+  let total = 0;
+  arr.map((item) => { total += item;})
+  return total;
+}
+
+const container = document.getElementById('amountToSell');
+container.textContent = availableForSale(availableStocks);
+
+// Opdracht 2A
+
+// const tvTypes = inventory.map((inventor) => {
+//   return inventor.type;
+// });
+//
+// console.log(tvTypes);
+// console.log('\n');
+
+// Opdracht 2B
+
+// const soldOut = inventory.filter((inventor) => {
+//   return (inventor.originalStock - inventor.sold) === 0;
+// });
+
+// console.log(soldOut);
+// console.log('\n');
+
+// Opdracht 2C
+
+// const ambiLight = inventory.filter((inventor) => {
+//   return inventor.options.ambiLight;
+// });
+
+// console.log('ALLE TV\'S MET AMBILIGHT: ');
+// console.log(ambiLight);
+// console.log('\n');
+
+// Opdracht 2D
+// console.log('Sorted by Price: ');
+// inventory.sort((a,b) => {return a.price - b.price});
+// // console.log(inventory);
+//
+// function sortByPrice(arr) {
+//   arr.sort((a,b) => {return a.price - b.price});
+//   return arr;
+// }
+
+// console.log(sortByPrice(inventory));
+
+// Opdracht 3A
+
+// console.log('PREDICTED REVENUE:');
+
+const predictedRevenue = inventory.map((inventor) => {
+  return (inventor.originalStock - inventor.sold) * inventor.price;
+});
+
+function totalPredictedRevenue(arr) {
+  let total = 0;
+  arr.map((item) => {total += item});
+  return total;
+}
+
+// console.log('€ ' + totalPredictedRevenue(predictedRevenue));
+// console.log('\n');
+
+const container2 = document.getElementById('projectedRevenue');
+container2.textContent = totalPredictedRevenue(predictedRevenue);
+
+// Opdracht 3B
+
+const revenueSoFar = inventory.map((inventor) => {
+  return (inventor.originalStock * inventor.price) - ((inventor.originalStock - inventor.sold) * inventor.price );
+});
+
+// console.log('REVENUE SO FAR:');
+
+function totalRevenueSoFar(arr) {
+  let total = 0;
+  arr.map((item) => {total += item});
+  return total;
+}
+
+// console.log('€ ' + totalRevenueSoFar(revenueSoFar));
+
+const container3 = document.getElementById('achievedRevenue');
+container3.textContent = totalRevenueSoFar(revenueSoFar);
+
+// Opdracht 4
+// const tvOne = document.getElementById('tvOne');
+// tvOne.textContent = inventory[0].type;
+//
```

Comment:
In het voorbeeld maken ze een list item aan in javascript `const tvOne = document.createElement('li')` en hier stoppen ze de beiden TV's in door middel van een `jouwVariabele.appendChild(tvOne)`